### PR TITLE
Setting the intiial velocity for a model or joint

### PR DESCRIPTION
**Pull request by chapulina**
# 🎉 New feature

## Summary

Let the user set an initial velocity through the `VelocityControl` plugin or `JointController` plugin.

I'm using this for a demo.

I'll come back to add tests and docs, just opening in time for feature freeze :cold_face: 

## Test it

I'll add it to the example world.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests: ~~**TODO**~~ Done
- [x] Added example and/or tutorial ~~**TODO**~~ Done
- [x] Updated documentation (as needed) ~~**TODO**~~ Done
- [ ] ~~Updated migration guide (as needed)~~
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

**Comment by ahcorde** [comment]
should we add any test ?

**Comment by chapulina** [comment]
Ok, tests, examples and docs added. Mind taking another quick look, @ahcorde ?

File: test/integration/velocity_control_system.cc [hunk 1]
```diff
@@ -136,6 +136,55 @@ TEST_P(VelocityControlTest, PublishCmd)
...
+      EXPECT_NE(kNullEntity, id);
+
+      auto poseComp = _ecm.Component<components::Pose>(id);
+      ASSERT_NE(nullptr, poseComp);
+
+      poses.push_back(poseComp->Data());
+    });
+  server.AddSystem(testSystem.systemPtr);
+
+  // Run server and check that vehicle didn't move
```
Comment (ahcorde) [inline test/integration/velocity_control_system.cc:170]:
```suggestion
  // Run server and check that vehicle move
```

**Review by ahcorde — APPROVED** [review]
A minor comment

**Pull request merged**